Modules/Server/Tests/PocketCastsServerTests/SyncTaskTests+BookmarkImport.swift

```diff
@@ -0,0 +1,181 @@
+@testable import PocketCastsServer
+import PocketCastsDataModel
+import FMDB
+import XCTest
+
+final class SyncTaskTests_BookmarkImport: XCTestCase {
+    private var dataManager: DataManager!
+    private var bookmarkManager: BookmarkDataManager!
+    private var syncTask: SyncTask!
+
+    override func setUp() {
+        dataManager = DataManager(dbQueue: FMDatabaseQueue(), shouldCloseQueueAfterSetup: false)
+        bookmarkManager = dataManager.bookmarks
+        syncTask = SyncTask(dataManager: dataManager)
+    }
+
+    // MARK: - Importing a single bookmark
+
+    func testNonexistingBookmarkIsAdded() async {
+        let uuid = UUID().uuidString
+        let episode = UUID().uuidString
+        let podcast = UUID().uuidString
+        let title = "Hello World"
+        let time = 86400.0
+        let created = Date(timeIntervalSince1970: 456)
+
+        let apiBookmark = Api_SyncUserBookmark.forTesting(uuid: uuid,
```

Comment:
Given we're already on the tests, I believe we can just initialize `Api_SyncUserBoomark` right away with the given parameters. We can just add this to your extension:

```swift
init(uuid: String,
                     episode: String = "episode",
                     podcast: String? = nil,
                     title: String = "Title",
                     time: TimeInterval = 1234,
                     created: Date = Date(),
                     isDeleted: Bool? = nil) {
        self.init()
        bookmarkUuid = uuid
        episodeUuid = episode

        if let podcast {
            podcastUuid = podcast
        }

        self.title.value = title
        self.time.value = Int32(time)
        createdAt = .init(date: created)

        if let isDeleted {
            self.isDeleted.value = isDeleted
        }
    }
```

Then we can remove the `.forTesting`